src/std/env.ab

```diff
@@ -74,3 +75,53 @@ pub fun is_root(): Bool {
 
     return false
 }
+
+pub fun printf(format: Text, args: [Text] = [""]): Null {
+    unsafe ${nameof args}=("{format}" "\$\{{nameof args}[@]}")$
+    unsafe $printf "\$\{{nameof args}[@]}"$
+}
+
+pub fun printf_escape(text: Text): Text {
+    return unsafe $echo \${nameof text}|sed -e 's/\\\\/\\\\\\\\/g' -e "s/%/%%/g"$
+}
+
+pub fun text_shell(message: Text, color1: Num, style: Num, color2: Num): Text {
+    return "\e[{style as Text};{color1 as Text};{color2 as Text}m{printf_escape(message)}\e[0m"
+}
```

Comment:
- `\033` octal escape is more broadly recognized by terminals
- No need to cast `Num` to `Text` in interpolation as it's done automatically
- `color1` is a foreground color so let's rename it to `fg`
- `color2` is a background color so let's rename it to `bg`
- Let's rearrange the values to match the same sequence

```suggestion
pub fun text_shell(message: Text, style: Num, fg: Num, bg: Num): Text {
    return "\033[{style};{fg};{bg}m{printf_escape(message)}\033[0m"
}
```

You know... we could add `enum` so that the `fg`, `bg` and `style` are just names and not numbers. But let's add that later.